components/twilio/sources/new-call/new-call.js

```diff
@@ -0,0 +1,29 @@
+const common = require("../common-webhook.js");
+
+module.exports = {
+  ...common,
+  key: "twilio-new-call",
+  name: "New Call (Instant)",
+  description:
+    "Configures a webhook in Twilio, tied to a phone number, and emits an event each time a call to that number is completed",
+  version: "0.0.1",
+  dedupe: "unique",
+  methods: {
+    ...common.methods,
+    getWebhookFn() {
+      return this.twilio.setIncomingCallWebhookURL.bind(this);
+    },
+    generateMeta(body, headers) {
+      return {
+        /** if Twilio retries a message, but we've already emitted, dedupe */
+        id: headers["i-twilio-idempotency-token"],
+        summary: `New call from ${body.From}`,
```

Comment:
**-Minor-**

This will expose phone numbers, which can present compliance issues for users.
I wrote [this method](https://github.com/PipedreamHQ/pipedream/blob/master/components/ringcentral/sources/common/base.js#L21) for a similar event source for RingCentral that might be of interest to you.

Reply:
Thanks! Updated.